api4/events.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package api4
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/mattermost/mattermost-server/v6/shared/eventbus"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
+func (api *API) InitEvents() {
+	// GET /api/v4/events/topics
+	api.BaseRoutes.Events.Handle("/topics", api.APISessionRequired(getTopics)).Methods("GET")
+	// GET /api/v4/events/topics/<topicId>
+	api.BaseRoutes.Events.Handle("/topics/{topic_id}", api.APISessionRequired(getTopic)).Methods("GET")
+	// GET /api/v4/events/schemas/<topicId>
+	api.BaseRoutes.Events.Handle("/schemas/{topic_id}", api.APISessionRequired(getSchema)).Methods("GET")
+}
+
+func getTopics(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Can you add a permission check for only manage system scheme? Let's start this for admins only. Do yo have any thoughts @oh6hay?